collection-hooks.js

```diff
@@ -136,12 +131,12 @@ CollectionHooks.extendCollectionInstance = function extendCollectionInstance (se
         } : self._hookAspects[method] || {},
         function (doc) {
           return (
-            isFunction(self._transform)
+            typeof self._transform === 'function'
             ? function (d) { return self._transform(d || doc) }
             : function (d) { return d || doc }
           )
         },
-        toArray(arguments),
+        Object.values(arguments),
```

Comment:
Can't we gather the args using a rest operator in the enclosing function and skip this Object.values altogether?

Reply:
Ok, I took on a bit of a broader modernization effort here: https://github.com/matb33/meteor-collection-hooks/pull/248